### PR TITLE
Feature - Update Find command to compared keywords with all fields

### PR DESCRIPTION
**Pull request by zihaowrez**
This commit fixes issue #68 

Changes:

- Updated the `NameContainsKeywordsPredicate` class so that the keywords are compared to all information of each person
- Renamed this class to `PersonContainsKeywordsPredicate`
- Test cases are updated accordingly

### CODE REVIEW

**Review by cheehongw — COMMENTED** [review]
Seems now the `Find` command will match keywords in both `name` and `Tags`, but is it possible or easy to extend the code for it to find a user given a matching phone number or any attribute of `Person`?

**Review by cheehongw — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/person/PersonContainsKeywordsPredicate.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+public class PersonContainsKeywordsPredicate implements Predicate<Person> {
+    private final List<String> keywords;
+
+    public PersonContainsKeywordsPredicate(List<String> keywords) {
+        this.keywords = keywords;
+    }
+
+    @Override
+    public boolean test(Person person) {
+        if (keywords.stream()
```
Comment (reignnz) [inline src/main/java/seedu/address/model/person/PersonContainsKeywordsPredicate.java:22]: Is it possible to abstract out the conditional statements into separate methods such that each one tests for name, phone, etc?

**Review by reignnz — CHANGES_REQUESTED** [review]
Other than that looks good to me!

**Pull request merged**